### DISCUSSION TITLE
"Step by Step"

### DISCUSSION THREAD

**Discussion by zavorra** [alf45tar/PedalinoMini#520] [Q&A]
Hi, 

I would like to use a "step by step" action on a sequence to emulate a latching button.

I defined my sequence and by action as follows

![Screenshot_20240612_120210](https://github.com/alf45tar/PedalinoMini/assets/12064434/69f52c9d-8f3b-43e5-b02f-4697751bd1b5)
![Screenshot_20240612_120257](https://github.com/alf45tar/PedalinoMini/assets/12064434/7b1c80b9-430b-47fb-a50c-a13444f7f78a)

but, no matter how I change the "step", "tag1", "tag2" parameters of the action, all I get on the debug serial is:
![image](https://github.com/alf45tar/PedalinoMini/assets/12064434/5cd7ad11-4c74-41d8-aa15-fb1df2ccf7f3)

What am I doing wrong?

**Comment by alf45tar** [comment]
If you want to give a try to version 3.4.2 it should be fixed.
